integration/go.mod

```diff
@@ -23,6 +23,6 @@ require (
 
 replace github.com/onflow/flow-go => ../
```

Comment:
Here I think it's ok to keep this at least in the short term 👍. Integration tests aren't really intended to be imported to other projects.